pkg/kubectl/describe.go

```diff
@@ -889,7 +889,8 @@ func describeJob(job *extensions.Job, events *api.EventList) (string, error) {
 		} else {
 			fmt.Fprintf(out, "Image(s):\t%s\n", "<no template>")
 		}
-		fmt.Fprintf(out, "Selector:\t%s\n", labels.FormatLabels(job.Spec.Selector))
+		selector, _ := extensions.PodSelectorAsSelector(job.Spec.Selector)
+		fmt.Fprintf(out, "Selector:\t%s\n", selector)
```

Comment:
I believe so but will test. From package "fmt" docs: If an operand implements method String() string, that method will be invoked to convert the object to a string, which will then be formatted as required by the verb (if any). String() string is defined on the Selector interface as a human readable string that represents this selector so should just work.